lib/IRGen/Linking.cpp

```diff
@@ -629,6 +629,8 @@ SILLinkage LinkEntity::getLinkage(ForDefinition_t forDefinition) const {
     return SILLinkage::Private;
 
   case Kind::DynamicallyReplaceableFunctionKey:
+    return getSILFunction()->getLinkage();
```

Comment:
@aschwaighofer This was the only actual change I had to make. Are you aware of anything that I might have missed?